buildpack.md

```diff
@@ -542,11 +551,19 @@ For each entry in `<plan>`:
 
 #### Bills-of-Materials
 
-When the build is complete, a BOM (Bill-of-Materials) describing the app image MAY be generated for auditing purposes.
-If generated, this BOM MUST contain all `bom` entries in each `launch.toml` at the end of each `/bin/build` execution, in adherence with the process and data format outlined in the [Platform Interface Specification](platform.md).
+Buildpacks MAY write standardized Bill of Materials (sBOM) files with extension `<ext>`, where `<ext>` MUST denote an sBOM media type based on Internet Assigned Numbers Authority (IANA) [assigned media types](https://www.iana.org/assignments/media-types/media-types.xhtml). The currently supported media types and their expected file extensions are as follows:
 
-When the build is complete, a build BOM describing the build container MAY be generated for auditing purposes.
-If generated, this build BOM MUST contain all `bom` entries in each `build.toml` at the end of each `/bin/build` execution, in adherence with the process and data format outlined in the [Platform Interface Specification](platform.md).
+ | sBOM Media Type                  | File Extension
+ |----------------------------------|----------------------------------------------
+ | `application/vnd.cyclonedx+json` | `cdx.json`
+ | `application/spdx+json`          | `spdx.json`
+ | `application/vnd.syft+json`      | `syft.json`
+
+When the build is complete, an sBOM describing the app image MAY be generated for auditing purposes.
+If generated, this BOM MUST contain all `<layer>.bom.<ext>` files for each `launch = true` layer at the end of each `/bin/build` execution, as well as `launch.bom.<ext>` if provided, in adherence with the process and data format outlined in the [Platform Interface Specification](platform.md).
+
+When the build is complete, a build sBOM describing the build container MAY be generated for auditing purposes.
+If generated, this BOM MUST contain all `<layer>.bom.<ext>` files for each `launch = false` layer at the end of each `/bin/build` execution, as well as `build.bom.<ext>` if provided, in adherence with the process and data format outlined in the [Platform Interface Specification](platform.md).
```

Comment:
1. Same comment as [this](https://github.com/buildpacks/spec/pull/268/files#r746061076) about leakiness.
2. Where do we talk about _data format_ in the platform spec? Do we mean file structure?

Reply:
@jromero for (1) above the link did not work. Are you talking about MUST vs MAY?

For (2), I'll change "data format" to "file structure". The old wording is a holdover from the old unstructured bom.

Reply:
Please see e79d4a0

Reply:
Ah re: (1) I see it here:

> Probably future refactor discussion here but...

> This is an interesting current leaky abstraction between the platform and buildpack APIs. As a platform implementor, I don't think I care about the contents of `<layers>/<buidpack-id>`. I wonder if this is the right spec to define this...

I agree, it's a bit weird. I guess it's helpful for the buildpack to know that the sBOM it writes is going "somewhere". We should bring this up as part of a larger spec refactor discussion. I would vote not to attempt a refactor as part of this PR.